test.hxml

```diff
@@ -0,0 +1,14 @@
+## Neko
+-main TestMain
+-lib munit
+-lib hamcrest
+-lib svg
+-lib openfl
+-lib lime
+
+-cp src
+-cp test
+-neko build/neko_test.n
+-D lime
+-D lime_cairo
+--remap flash:openfl
```

Comment:
I don't really understand how this works.

I moved the `neko` part to the end. `-neko build/neko_test.n` works but `-cmd neko build/neko_test.n` doesn't seem to do anything.